### ISSUE TITLE
when not host - white box appearing in garrison when mousing over

### ISSUE THREAD

**Issue by dotdotdot-tts** [rogeriocamorim/duneBugSuggestionTracker#47]
surprised this hasn't been reported yet, i've been lazy logging into github haha. this has been a complaint in almost every game i've played since the last update.

this seems to only appear when you're **not host**, a white box appears in garrison when mousing over troops. makes clicking/moving troops difficult to do.

**Comment by BShadow993** [comment]
I have noticed this as well, lots of folks are having trouble choosing individual cubes in garrisons. Is noticeable on videos and streams.

**Comment by Chatanga** [comment]
As I read it: some players see a white cubic outline around a troop on which they hover their mouse (perfectly normal), try to left/right click on it, notice that it does nothing at all (once again, perfectly normal) and end up complaining about it. For the record, you have to move your mouse directly over a garrison for the left/right click to work.

**Comment by dotdotdot-tts** [comment]
the white box issue was not present in previous versions.

it is not a visual issue, somehow it does make picking up troops out of the garrison more difficult to do.

i did not specify about right/left clicking the garrison. there are no issues with that. 

blackshadow corroborated the issue; it never happens to host, just the other players.

**Comment by Chatanga** [comment]
I can't reproduce the problem in multiplayer. Don't you have a screenshot of this mysterious white box?

**Comment by dotdotdot-tts** [comment]
i will tomorrow, thank you. 

i have to join a game so i am not host. it will not appear when you are the host.

**Comment by dotdotdot-tts** [comment]
screenshot below, this is me picking up one troop and trying to pick up another, but the white box is in the way:
![troop trying to pick up second](https://github.com/rogeriocamorim/duneBugSuggestionTracker/assets/154347378/dfa54dfa-3016-465c-bdd3-f23938d00a0a)

**Comment by dotdotdot-tts** [comment]
you can eventually pick it up but it's really annoying, this wasn't ever present in previous versions.

**Comment by Chatanga** [comment]
Thanks for the screenshot dotdotdot. The white box is actually the underlying garrison anchor that is hovered over (ctrl + alt + click on the colored area to select it). I don't see how this would interfere with troop selection, but apparently it does. Since these are the only anchors that are interactable (I don't know why I defined them that way), I simply made then non-interactable. I hope this resolves the issue. The weird thing however is that garrisons have always been like this, no changes have been made...

Fixed (I hope) in: [Got rid of the weird customisation of the garrison anchors.](https://github.com/aaittouares/Dune_Uprising_TTS/commit/6b1d48daf5bca6c0dac40ad511dade3d773264ab)
